<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>五子棋</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/res/ICON.ico</normaloff>:/images/res/ICON.ico</iconset>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>640</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/res/mainmenu.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnDoubleGame">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent no-repeat;
background-image:url(:/images/res/doubleButton.png);}
QPushButton:hover{background:transparent no-repeat;
background-image:url(:/images/res/doubleButton_h.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnManMachine">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent no-repeat;
background-image:url(:/images/res/singleButton.png);}
QPushButton:hover{background:transparent no-repeat;
background-image:url(:/images/res/singleButton_h.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnOnline">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>400</y>
      <width>124</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent no-repeat;
background-image:url(:/images/res/netButton.png);}
QPushButton:hover{background:transparent no-repeat;
background-image:url(:/images/res/netButton_h.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnExit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>460</y>
      <width>60</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent no-repeat;
background-image:url(:/images/res/exitButton.png);}
QPushButton:hover{background:transparent no-repeat;
background-image:url(:/images/res/exitButton_h.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnHelp">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>60</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent no-repeat;
background-image:url(:/images/res/helpButton.png);}
QPushButton:hover{background:transparent no-repeat;
background-image:url(:/images/res/helpButton_h.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
